File: CSharp/k01_3.cs
using System;

namespace CSharp
{
    public class K01_3 : IExecutable
    {
        public void Execute()
        {
            Console.WriteLine("整数：" + 12345);
            Console.WriteLine("実数：" + 123.456789);
            Console.WriteLine("文字：" + 'X');
            Console.WriteLine("文字列：" + "ABCdef");
        }
    }
}

File: Java/src/com/ecc/gkgo1a/K01_3.java
package com.ecc.gkgo1a;

public class K01_3 implements IExecutable {

    @Override
    public void Execute()
    {
        System.out.println("整数：" + 12345);
        System.out.println("実数：" + 123.456789);
        System.out.println("文字：" + 'X');
        System.out.println("文字列：" + "ABCdef");
    }
}
